inst/rmarkdown/templates/hugo/resources/default.md

```diff
@@ -7,6 +7,9 @@ title: $if(title)$$title$$endif$
 categories: Data Science
 tags: 
   - R
+  - $if(tag1)$$tag1$$endif$
+  - $if(tag2)$$tag2$$endif$
+image: $if(image)$$image$$endif$
```

Comment:
👍   Works well.  It is important that tags be last though - if there is an empty line with tag2 not present and image comes after, it breaks Hugo.